crypto/armcap.c

```diff
@@ -168,12 +153,30 @@ void OPENSSL_cpuid_setup(void)
             if (hwcap & HWCAP_CE_SHA256)
                 OPENSSL_armcap_P |= ARMV8_SHA256;
 
-# ifdef __aarch64__
+#  ifdef __aarch64__
             if (hwcap & HWCAP_CE_SHA512)
                 OPENSSL_armcap_P |= ARMV8_SHA512;
-# endif
+#  endif
         }
-    } else if (sigsetjmp(ill_jmp, 1) == 0) {
+        return;
```

Comment:
Hmm, now when I see it would need to continue to the ARMV7_TICK.
I still feel like it was not a good idea to call getauxval with the SIGILL trap in place.
That should only be done for assembler code we control.